ocpmodels/models/base.py

```diff
@@ -29,8 +29,14 @@ def __init__(self, num_atoms=None, bond_feat_dim=None, num_targets=None):
     def forward(self, data):
         raise NotImplementedError
 
-    def generate_graph(self, data):
-        if not self.otf_graph:
+    def generate_graph(
+        self, data, otf_graph=None, cutoff=None, max_neighbors=None
+    ):
+        otf_graph = otf_graph or self.otf_graph
+        cutoff = cutoff or self.cutoff
+        max_neighbors = max_neighbors or self.max_neighbors
+
```

Comment:
Should we also add `use_pbc` here? That would just complete all required settings.